cfme/fixtures/nuage.py

```diff
@@ -6,46 +8,64 @@
 
 
 def create_basic_sandbox(nuage):
-    box = {}
+    box = Namespace()
 
     # Create empty enterprise aka 'sandbox'.
-    enterprise = box['enterprise'] = nuage.create_enterprise()
-    logger.info('Created sandbox enterprise %s (%s)', enterprise.name, enterprise.id)
+    box.enterprise = get_object_dictionary(nuage.create_enterprise())
+    logger.info('Created sandbox enterprise %s (%s)',
+                box.enterprise['obj'].name, box.enterprise['obj'].id)
 
     # Fill the sandbox with some entities.
     # Method `create_child` returns a tuple (object, connection) and we only need object.
-    box['template'] = enterprise.create_child(
-        nuage.vspk.NUDomainTemplate(name=random_name()))[0]
-    box['domain'] = enterprise.create_child(
-        nuage.vspk.NUDomain(name=random_name(), template_id=box['template'].id))[0]
-    box['zone'] = box['domain'].create_child(
-        nuage.vspk.NUZone(name=random_name()))[0]
-    box['subnet'] = box['zone'].create_child(
+    box.template = get_object_dictionary(box.enterprise['obj'].create_child(
```

Comment:
I don't think this layer-cake of Namespace/attribute/dictionary/key makes this entity generation very readable. I'll keep reading, but I'm continuing to think this `get_object_dictionary` layer is a bad pattern.